CONTRIBUTING.md

```diff
@@ -35,14 +35,14 @@ For Git commit messages, our principle is that `git log --pretty=oneline` should
 
 The following is an example of a well-formed commit, including the description (first line) and a body that includes changelog additions:
 
-      Introduced a new API for package management (#1311)
+      Fixes #1311
 
-      Also fixes a typo in the Scala bindings documentation
+      Also fixes a typo in the Scala bindings documentation.
 
       CHANGELOG_BEGIN
 
       - [Sandbox] Introduced a new API for package management.
-      See `#1311 <https://github.com/digital-asset/daml/issues/1311>`__.
+        See `#1311 <https://github.com/digital-asset/daml/issues/1311>`__.
 
       CHANGELOG_END
```

Comment:
I have no idea why this document uses 6 spaces for indenting code, but it really confuses GitHub. I suggest changing it to use 4 spaces.

Reply:
as above